data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -22,8 +22,8 @@
 
 <node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
   <!--
-      org.freedesktop.portal.GlobalShortcut:
-      @short_description: GlobalShortcut portal backend interface
+      org.freedesktop.portal.GlobalShortcuts:
+      @short_description: GlobalShortcuts portal backend interface
```

Comment:
Since you're at it: `backend` → `frontend`

Reply:
Took the liberty to change the entire sentence to be more in line with existing portal frontends.